lms/static/scripts/frontend_apps/components/dashboard/StudentsActivity.tsx

```diff
@@ -49,20 +49,20 @@ export default function StudentsActivity() {
             students.error ? 'Could not load students' : 'No students found'
           }
           title={assignment.isLoading ? 'Loading...' : title}
+          rows={orderedStudents}
           columns={[
-            { field: 'display_name', label: 'Name', classes: 'w-[60%]' },
+            { field: 'display_name', label: 'Student', classes: 'w-[60%]' },
```

Comment:
Switching from "Name" to "Student", so that it's more obvious what are the items in the list.

Reply:
Will we ever list non-student annotations in the list?

Reply:
Could be, but I'd suggest we think about the right title or the right way to determine the title then.